### PR TITLE
Refactoring

### PR DESCRIPTION
**Pull request by avbel**
Added support 1-click deploy to Heroku.
Reduced amount of settings (like `baseUrl`)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,26 +1,43 @@
 #node-voice-reference-app
 
-  This application demonstrates how to implement voice calling for mobile devices, browsers (WebRTC), and any SIP client using the Bandwidth
-    Application Platform.
+  This application demonstrates how to implement voice calling for mobile devices, browsers (WebRTC), and any SIP client using the [Catapult API](http://ap.bandwidth.com/?utm_medium=social&utm_source=github&utm_campaign=dtolb&utm_content=_).
     This reference application makes creating, registering, and implementing voice calling for endpoints (mobile, web, or any SIP client) easy.
     This application implements the steps documented [here](http://ap.bandwidth.com/docs/how-to-guides/use-endpoints-make-receive-calls-sip-clients/).
```
Comment (dtolb) [inline README.md]: can you update this link as well /?utm_medium=social&utm_source=github&utm_campaign=dtolb&utm_content=_

File: README.md [hunk 2]
```diff
@@ -1,26 +1,43 @@
...
+
+## Deploy To PaaS
+
+#### Env Variables Required To Run
+* ```CATAPULT_USER_ID```
+* ```CATAPULT_API_TOKEN```
+* ```CATAPULT_API_SECRET```
+* ```CATAPULT_DOMAIN_NAME```
+
+[![Deploy](https://www.herokucdn.com/deploy/button.svg)](https://heroku.com/deploy)
```
Comment (dtolb) [inline README.md]: :+1:

File: app.json [hunk 3]
```diff
@@ -0,0 +1,22 @@
+{
+  "name": "Hybrid Catapult/Iris app",
```
Comment (dtolb) [inline app.json]: Name?

File: app.json [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+  "description": "It demonstrates how to implement voice calling for mobile devices, browsers (WebRTC), and any SIP client using Catapult API.",
```
Comment (dtolb) [inline app.json]: description

**Comment by dtolb** [comment]
Found a few small tweaks but looks good!

File: README.md (hunk 1 above)
Comment (avbel) [inline README.md]: done

File: app.json (hunk 4 above)
Comment (avbel) [inline app.json]: Description was right

**Comment by avbel** [comment]
Done

**Pull request merged**